src/otx/algo/utils/mmconfig.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Utils used for MMConfigs."""
+
+import inspect
+from pathlib import Path
+
+from omegaconf import DictConfig, OmegaConf
+
+
+def read_mmconfig(model_name: str, subdir_name: str = ".") -> DictConfig:
+    """Read MMConfig.
+
+    It try to read MMConfig from the yaml file which exists in
+    `<Directory path of __file__ who calls this function>/mmconfigs/<subdir_name>/<model_name>.yaml`
+    """
+    frame = inspect.stack()[1]
+    module = inspect.getmodule(frame[0])
+
+    if module is None or (mod_fpath := module.__file__) is None:
+        msg = "Cannot get valid model from stack"
+        raise RuntimeError(msg)
+
+    root_dir = Path(mod_fpath).parent / "mmconfigs" / subdir_name
+    yaml_fpath = root_dir / f"{model_name}.yaml"
+
+    if not yaml_fpath.exists():
+        raise FileNotFoundError
```

Comment:
can you add a message to specify which file is missing. It would be easier to debug this way.

Reply:
Fixed in https://github.com/openvinotoolkit/training_extensions/pull/2776/commits/669d935209ac14f857b52b25ce3957fc9ce03f15